srcpkgs/drumstick/template

```diff
@@ -0,0 +1,32 @@
+# Template file for 'drumstick'
+pkgname=drumstick
+version=1.1.2
+revision=1
+build_style=cmake
+configure_args="-DLIB_SUFFIX="
+hostmakedepends="pkg-config doxygen libxslt docbook-xsl"
+makedepends="qt5-svg-devel alsa-lib-devel pulseaudio-devel"
+short_desc="a set of MIDI libraries using C++/Qt5 idioms and style"
+maintainer="Mohammad Amin Sameti <mamins1376@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="http://${pkgname}.sourceforge.net"
```

Comment:
shorter to say "drumstick" than "${pkgname}".

Reply:
Also, this is personal preference, but I prefer to not have variables in the homepage. It makes it harder to check the homepage when listed in `void-updates`.